app/views/layouts/base.html.erb

```diff
@@ -60,6 +60,19 @@
     <%= vite_client_tag %>
     <%= vite_javascript_tag 'application' %>
 
+    <% if AnalyticsService.analytics_events.present? && @current_user&.collect_analytics? %>
+      <script data-analytics-events>
+        window.dataLayer = window.dataLayer || [];
+
+        <% AnalyticsService.analytics_events.each do |event| %>
+            window.dataLayer.push({
+              event: "event_data",
```

Comment:
Why is the event called `event_data`?

Reply:
And we use `event_data` for all our custom events? If we have multiple events called `event_data` does the event data get coalesced by GA into one event?

Reply:
I don't think the events get coalesced at any point and I think they can always be distinguished by name.

I'm not that up on how these are documented or at where @AnneCarr might be able to help. Originally there was talk of a single data environment shared across *.gov.uk properties. I don't know any specifics though.